src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.0.0
+September 1, 2022
+
+### Removal Notice
+- Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md) is no longer supprted.
+
+### Deprecation Notice
+- Support for Kubernetes [1.20](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md) has been derprecated and will be removed next release.
+
+### Features
+- License enforcements changed to allow for easier adoption.
+    - Unlicensed instances will be limited to 1 user.
+    - Instances with a trial license will be limted to 3 users.
+    - Scale and Enterprise limits will be softly enforced to ease user management and promote adoption of your installation.
+- Okteto UI now allows users to specify the manifest path when launching a dev environment.
+
+
+### Improvements
+- Chart has been renamed from "Okteto Enterprise" to "Okteto". This will result in all kubernetes resources being renamed in the okteto namespace.
+- Installation has been simplified through improvements in certificate management.
+    - The cert-manager chart is no long bundled with Okteto and is no longer installed.
+    - Okteto will now use a self-signed certificate by default.
+- Add support for Kubernetes [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md).
+- Upgrade ingress-nginx to [4.2.1](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.2.1).
+- Exposed grpc connection configuration for the buildkit grpc client and server.
+
+### Bugfixes
+- Fixed an issue where PVCs were not deleting if volumeSnapshots were enabled.
```

Comment:
Maybe there's a typo? "Were not deleted"

Reply:
- [ ] reword for clarity